frontend/src/components/pages/camps/CampersTableOverviewIconGroup.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+
+import { Text, Image, HStack } from "@chakra-ui/react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faPerson,
+  faHandDots,
+  faHandshakeAngle,
+} from "@fortawesome/free-solid-svg-icons";
+
+import icon_sunrise from "../../../assets/icon_sunrise.svg";
+import icon_sunset from "../../../assets/icon_sunset.svg";
+
+import { Camper } from "../../../types/CamperTypes";
+
+const TableOverviewIcon = ({
+  customIcon,
+  icon,
+  description,
+  color,
+}: {
+  customIcon: boolean;
+  icon: any;
+  description: string;
+  color: string;
+}) => {
+  return (
+    <HStack
```

Comment:
These are filters, so they will need an onclick function which would further filter out the results of the table. See the following specs on the dev handoff page:
<img width="863" alt="image" src="https://user-images.githubusercontent.com/57159970/186127788-318c83b8-3b06-449e-834c-77ecbae893c5.png">

Note that we don't need the camera and laptop filters (as well as the statuses for them in the table rows)